content/log/explore.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Explore
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+<div class="alert alert-info">
+Datadog's log management solution is is currently in private beta. If you would like to apply to it, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
```

Comment:
To me, "solution" sounds stale and enterprise-y. How about just "Datadog log management"?

Reply:
👍